.travis/retry-command.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 
 function retry_command {
-    local usage="Usage: ${FUNCNAME[0]} [-n numtries] [-d delay]"
+    local usage="Usage: ${FUNCNAME[0]} [-b] [-d delay] [-n numtries]"
     local OPTIND OPTION
-    local numtries=3 delay=2
+    local backoff_factor=2 delay=3 numtries=4
 
-    while getopts ":n:d:" OPTION; do
+    while getopts ":bd:n:" OPTION; do
         case "${OPTION}" in
-        n)
-            numtries=${OPTARG}
+        b)
+            backoff_factor=1
```

Comment:
So, by default we double the time between each retry, but if we pass `-b` we don't back off? That seems less than obvious. Maybe make `-b` take an option for the factor?

Reply:
I considered doing it that way originally. But I've done it now.